### ISSUE TITLE
throwing items a single tile doesn't  work

### ISSUE THREAD

**Issue by lm40** [VOREStation/VOREStation#11648]
if you throw an item to an adjacent tile, the item appears on the adjacent tile, but ALSO remains in your hand.

**Comment by Rykka-Stormheart** [comment]
Yes, caused by https://github.com/VOREStation/VOREStation/pull/11560

It was reverted on CHOMP and has since resolved that issue, afaik? I'll confirm.

**Comment by lm40** [comment]
ehh it is an easy fix regardless
